### ISSUE TITLE
Table resizable columns

### ISSUE THREAD

**Issue by permafrost06** [permafrost06/caseDB#24]
The following to be addressed for the table component:

1. The table columns to become resizable when the component is passed a prop.
```vue
<script setup>
const resize = ref(true);
</script>

<template>
    <TableComponent :resizable="resize" />
</template>
```

2. The resizer element should follow the mouse cursor. The mouse cursor should not get away from the resizer.
3. Resizing one column should only affect the widths of the 2 columns adjacent to the resizer.

**Comment by permafrost06** [comment]
@protibimbok take a look please

**Comment by permafrost06** [comment]
This issue can be closed once 
>The table columns to become resizable when the component is passed a prop.

has been addressed. Pinging @protibimbok

**Comment by protibimbok** [comment]
I was waiting for the other branches to be merged...

**Comment by permafrost06** [comment]
closed by #37

**Comment by permafrost06** [comment]
Closed by #37
